smart_contract/utils/src/debug.rs

```diff
@@ -23,18 +26,23 @@ mod host {
 /// Print `obj` in debug representation to the stdout.
 ///
 /// Do nothing if `debug` feature is not specified
```

Comment:
Docs are not correct now, because we print in any case if `target_family = "wasm"`

Reply:
True, I'll update those